### PR TITLE
Master

### PR DESCRIPTION
**Pull request by OJDUDE**
Templates validation errors fixes...

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@OJDUDE__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (omjaber). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by OJDUDE** [comment]
This template requires a pre-setup as it operates in existing VM and does not create one. We really should have an option in test to skip. Can someone from the Azure team review it please.

File: 100-sqlvm-setupkeyvault/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "virtualMachineName": {
```
Comment (singhkay) [inline 100-sqlvm-setupkeyvault/azuredeploy.json]: Rename this to existingVirtualMachineName to set the right expectation i.e. this is a VM that needs to already be created when executing this template and will not be created by this template

File: 100-sqlvm-setupkeyvault/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# This template setup Azure Key Vault on any existing Azure Virtual machine with SQL Server Standard or Enterprise edition.
```
Comment (singhkay) [inline 100-sqlvm-setupkeyvault/README.md]: Please rename the template to 101-sql-vm-keyvault-setup for better readability. Another options is to rename to 101-vm-sql-keyvault-setup so it fits in with the other 101-vm templates in the repo.

**Pull request closed without merging**